lib/hero-image-header/test/index.test.js

```diff
@@ -6,7 +6,7 @@ import Header from '../';
 describe('Component', function () {
   describe('<Header />', function () {
     it('Should render Header component', function (done) {
-      const wrapper = shallow(<Header />);
+      const wrapper = shallow(<Header displayedItems={[]}/>);
       expect(wrapper.find('.headerBarWrapper')).to.have.length(1);
```

Comment:
You could also easily add a test that checks the arrow is displayed when the displayedItems array has items!